mlflow/server/__init__.py

```diff
@@ -65,5 +66,6 @@ def _run_server(file_store_path, default_artifact_root, host, port, workers, sta
     if static_prefix:
         env_map[STATIC_PREFIX_ENV_VAR] = static_prefix
     bind_address = "%s:%s" % (host, port)
-    exec_cmd(["gunicorn", "-b", bind_address, "-w", "%s" % workers, "mlflow.server:app"],
+    opts = gunicorn_opts.split(" ") if gunicorn_opts else []
+    exec_cmd(["gunicorn"] + opts + ["-b", bind_address, "-w", "%s" % workers, "mlflow.server:app"],
```

Comment:
Out of curiosity, what happens if the user-specified `gunicorn_opts` contains a bind address or worker specification?

Reply:
good question. Can we target this as another PR. Maybe we can swap out `subprocess` with something like this http://docs.gunicorn.org/en/latest/custom.html

Reply:
Sounds good! Can you file an issue to make sure that this is tracked?

Reply:
done #630